cypress/utils/commands.js

```diff
@@ -676,3 +683,28 @@ Cypress.Commands.add('removeSampleDataFromDataSource', (dataSourceTitle) => {
     .should('be.visible')
     .click();
 });
+
+Cypress.Commands.add('createDataSourceNoAuthWithTitle', (title) => {
```

Comment:
can we move this to the utils file under data source management for better structure?

Reply:
updated the code, and already move this under data_source_management plugin